focuspoints.lrdevplugin/AppleDelegates.lua

```diff
@@ -0,0 +1,70 @@
+--[[
+  Copyright 2017 philmoz & Joshua Musselwhite, Whizzbang Inc
```

Comment:
I actually don't know enough about licensing to know if this is legit. I'm not sure if the license needs to stay the same or not. Looking through other Github repos, I haven't found any where individual contributors are listed in the license at a later time. I need to look more into this.